### PR TITLE
Public home component schema updated, public page body cleaned

### PR DESCRIPTION
**Pull request by mostafizur99**
## Issue Checklist with Issue IDs:

- [x] home page data props replaced with block
- [x] BlogCard block data schema updated
- [x] ReviewCard block data schema updated
- [x] JobCard block data schema updated
- [x] CategoryCard block data updated
- [x] BannerOne schema updated
- [x] search component searchByCategory field added
- [x] dashboard my-profile page body removed
- [x] public home body removed

## Logged Hours
[Submitted By](https://github.com/mostafizur99)
[Logged Hours](#)
[Submission Date](#)

### CODE REVIEW

File: apps/site/src/app/(private)/dashboard/(multi)/my-profile/page.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { redirect } from "next/navigation"
-import ProfilePageBody from "./PageBody"
 import { auth } from "@/context/auth"
+import { MyProfile } from "@padma/metajob-ui"
```
Comment (zelal-dev) [inline apps/site/src/app/(private)/dashboard/(multi)/my-profile/page.tsx:3]: ### Issue: Hardcoded Import

 **Description**
We cannot import like this: `import { MyProfile } from "@padma/metajob-ui"`.  
We need to ensure that any theme can be used. Avoid hardcoding `@padma/metajob-ui`.

File: apps/site/src/app/(public)/page.tsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
+import React, { Fragment } from "react"
+import type { Metadata } from "next"
 import { find } from "@/lib/strapi"
 import { StrapiSeoFormate } from "@/lib/strapiSeo"
-import type { Metadata } from "next"
-import DynamicBlockRenderer from "./body"
-import React from "react"
 import { getLanguageFromCookie } from "@/utils/language"
+import { getPublicComponents } from "@padma/metajob-ui"
```
Comment (zelal-dev) [inline apps/site/src/app/(public)/page.tsx:6]: ### Issue: Hardcoded Import

 **Description**
We cannot import like this: `import { MyProfile } from "@padma/metajob-ui"`.  
We need to ensure that any theme can be used. Avoid hardcoding `@padma/metajob-ui`.

**Review by zelal-dev — COMMENTED** [review]
@mostafizur99  I need rules and regulation list to review any code and accept any PR

File: apps/site/src/app/(public)/page.tsx (hunk 2 above)
Comment (mostafizur99) [reply apps/site/src/app/(public)/page.tsx:6]: This import will be updated in the next updated PR

**Pull request merged**